docs/src/main/asciidoc/extension-writing-dev-service.adoc

```diff
@@ -0,0 +1,123 @@
+////
+This document is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="extension-writing-dev-service"]
+= Writing a dev service
```

Comment:
We tend to use `Dev Service` as a brand.

Reply:
Will update, thanks.

Reply:
Sorry if it wasn't clear by we should be consistent with using `Dev Service` in the whole document.

If you can all squash when done, that would be perfect!

Reply:
Sorry, done now! I also made the change across a bunch of other documents, for consistency, so the changeset is rather bigger. :)